src/ch05-03-method-syntax.md

```diff
@@ -111,62 +111,61 @@ provide.
 > (&p1).distance(&p2);
 > ```
 >
-> The first one looks much cleaner. This automatic referencing behavior works
-> because methods have a clear receiver—the type of `self`. Given the receiver
-> and name of a method, Rust can figure out definitively whether the method is
-> reading (`&self`), mutating (`&mut self`), or consuming (`self`). The fact
-> that Rust makes borrowing implicit for method receivers is a big part of
-> making ownership ergonomic in practice.
+> Pierwszy wygląda o wiele bardziej przejrzyście. Zastosowana w niej jest automatyczna referencja,
+> ponieważ metoda ma wyraźnie oznaczonego odbierającego (*ang. receiver*) - typ `self`.
+> Mając informacje o odbierającym oraz o nazwie metody Rust jest w stanie jednoznacznie
+> stwierdzić, czy metoda odczytuje (`&self`), zmienia (`&mut self`) lub konsumuje (`self`).
+> Wymaganie przez Rusta oznaczenia pożyczania w odbierającym metody jest
+> w dużej części dlaczego mechanizm posiadania jest tak ergonomiczny w używaniu.
 
-### Methods with More Parameters
+### Metody z wieloma parametrami
 
-Let’s practice using methods by implementing a second method on the `Rectangle`
-struct. This time, we want an instance of `Rectangle` to take another instance
-of `Rectangle` and return `true` if the second `Rectangle` can fit completely
-within `self`; otherwise it should return `false`. That is, we want to be able
-to write the program shown in Listing 5-14, once we’ve defined the `can_hold`
-method.
+Poćwiczymy używanie metod implementując drugą metodę na strukturze `Rectangle`.
+Tym razem chcemy, żeby instancja struktury `Rectangle` przyjęła inną instancję `Rectangle`,
+i zwróciła: wartość `true` (*ang. prawda*), jeśli ta inna instancja `Rectangle`
```

Comment:
"wartość `true` (*ang. prawda*)" - odpuściłabym to w nawiasie ;)

Reply:
ok zmienione